pkgs/applications/misc/st/default.nix

```diff
@@ -15,6 +15,9 @@ stdenv.mkDerivation rec {
   inherit patches;
 
   configFile = optionalString (conf!=null) (writeText "config.def.h" conf);
+  patchPhase = optionalString stdenv.isDarwin ''
```

Comment:
Won't this prevent the `patches` list from being applied on darwin?

Reply:
Oops, yes